src/libsync/owncloudpropagator.cpp

```diff
@@ -410,6 +410,12 @@ quint64 OwncloudPropagator::smallFileSize()
     return smallFileSize;
 }
 
+quint64 OwncloudPropagator::jobTimeoutDuration()
```

Comment:
You don't need this function. And that's not even a good name. (you don't mention it is for abort, you don't mention what unit it is)

Since it is only used once, just have the constant in OwncloudPropagator::abort

(Note: in new code, the time constant should probably be in Ideally, in new code, the type of such constant should be `std::chrono::milliseconds` so the unit is in the type )